src/plugins/event-plugins/JsErrorPlugin.ts

```diff
@@ -6,14 +6,17 @@ export const JS_ERROR_EVENT_PLUGIN_ID = 'com.amazonaws.rum.js-error';
 
 export type PartialJsErrorPluginConfig = {
     stackTraceLength?: number;
+    shouldIgnoreError?: (error: ErrorEvent) => boolean;
 };
 
 export type JsErrorPluginConfig = {
     stackTraceLength: number;
+    shouldIgnoreError?: (error: ErrorEvent) => boolean;
 };
 
 const defaultConfig: JsErrorPluginConfig = {
-    stackTraceLength: 200
+    stackTraceLength: 200,
+    shouldIgnoreError: () => false
```

Comment:
Shouldn't need a default as it is not required

Reply:
It's not clear to me as to why default values are not required for optional parameters. Is it anti-pattern? I thought it makes sense to have a default value in case the user does not want to pass in anything (seems to be what we currently have in Orchestration as well).